sw/airborne/autopilot.c

```diff
@@ -270,10 +270,13 @@ void autopilot_set_power_switch(bool power_switch)
     gpio_clear(POWER_SWITCH_GPIO);
   }
 #endif
+
+#ifndef FIX_HACK_FREEK
```

Comment:
Something accidentally ended up in a PR which should have been added.
You can change this to `#if USE_CHIBIOS_RTOS`. That would properly fix it.

Reply:
Oh wait This is our own repo ^^